rom/dev/src/flow/cold_reset/idev_id.rs

```diff
@@ -36,6 +40,15 @@ const DOE_IV: Array4x4 = Array4xN::<4, 16>([0xfb10365b, 0xa1179741, 0xfba193a1,
 /// Maximum Certificate Signing Request Size
 const MAX_CSR_SIZE: usize = 512;
 
+const DUAL_CERT_MODE: bool = true;
```

Comment:
Do we need to add a TODO for this to be read from a fuse or elsewhere?

Reply:
I think we'll want it to be read from a register, but +1

Reply:
Resolving as this no longer applies.